src/node_kernels/TKESSTTAMSNodeKernel.C

```diff
@@ -0,0 +1,75 @@
+/*------------------------------------------------------------------------*/
+/*  Copyright 2019 National Renewable Energy Laboratory.                  */
+/*  This software is released under the license detailed                  */
+/*  in the file, LICENSE, which is located in the top-level Nalu          */
+/*  directory structure                                                   */
+/*------------------------------------------------------------------------*/
+
+#include "node_kernels/TKESSTTAMSNodeKernel.h"
+#include "Realm.h"
+#include "SolutionOptions.h"
+
+#include "stk_mesh/base/MetaData.hpp"
+#include "utils/StkHelpers.h"
+#include <SimdInterface.h>
+
+namespace sierra {
+namespace nalu {
+
+TKESSTTAMSNodeKernel::TKESSTTAMSNodeKernel(
+  const stk::mesh::MetaData& meta, const std::string coordsName)
+  : NGPNodeKernel<TKESSTTAMSNodeKernel>(),
+    dualNodalVolumeID_(get_field_ordinal(meta, "dual_nodal_volume")),
+    coordinatesID_(get_field_ordinal(meta, coordsName)),
+    viscID_(get_field_ordinal(meta, "viscosity")),
+    tviscID_(get_field_ordinal(meta, "turbulent_viscosity")),
+    tkeNp1ID_(get_field_ordinal(meta, "turbulent_ke", stk::mesh::StateNP1)),
+    sdrNp1ID_(get_field_ordinal(
+      meta, "specific_dissipation_rate", stk::mesh::StateNP1)),
+    prodID_(get_field_ordinal(meta, "average_production")),
+    densityID_(get_field_ordinal(meta, "average_density")),
```

Comment:
Again.. only difference from SST is in use of `average_density` and `average_production`?

Reply:
yes, that is correct